packaging/windows/generatemsi.ps1

```diff
@@ -124,39 +124,33 @@ $WixRoot = AcquireWixTools
 
 if([string]::IsNullOrEmpty($WixRoot))
 {
-    return -1
+    Exit -1
 }
 
 if(-Not (RunHeat))
 {    
-    return -1
+    Exit -1
 }
 
 if(-Not (RunCandle))
 {
-    return -1
+    Exit -1
 }
 
 if(-Not (RunLight))
 {
-    return -1
+    Exit -1
 }
 
 if(!(Test-Path $DotnetMSIOutput))
 {
     throw "Unable to create the dotnet msi."
-    return -1
+    Exit -1
 }
 
 Write-Host -ForegroundColor Green "Successfully created dotnet MSI - $DotnetMSIOutput"
 
-& $PSScriptRoot\testmsi.ps1 -inputMsi $DotnetMSIOutput
-
-if($LastExitCode -ne 0)
-{
-    Write-Host -ForegroundColor Red "Msi testing failed."
-    Exit 1
-}
+_ $PSScriptRoot\testmsi.ps1 @("$DotnetMSIOutput")
```

Comment:
It should. The documentation here is a bit loose, but failure is consistent with my experiments around non-0 exit codes.